src/app/fdctl/utility.c

```diff
@@ -270,7 +283,7 @@ snprintf1( char * s,
 }
 
 void
-self_exe( char * path ) {
+current_executable_path( char path[ PATH_MAX ] ) {
```

Comment:
```suggestion
current_executable_path( char path[ static PATH_MAX ] ) {
```
I think that's the syntax. Richie showed me. `static` makes the compiler actually check it.